src/backend/web/routes/log.js

```diff
@@ -0,0 +1,42 @@
+require('../../lib/config');
+const express = require('express');
+const path = require('path');
+const fs = require('fs').promises;
+
+const log = express.Router();
+
+log.get('/', async (req, res) => {
+  if (process.env.LOG_FILE) {
+    try {
+      const logs = await fs.readFile(
+        path.join(__dirname, '../../../../', process.env.LOG_FILE),
```

Comment:
Let's require the log file to be an absolute path instead of doing this `../..` step, as it's going to be brittle if anything moves around in our code.  If we crash trying to open your log file, you can fix your log file path.